### PR TITLE
Add Viberate ICO

### PR DESCRIPTION
**Pull request by MostafaBalata**
screen shot 2017-09-13 at 2 10 38 pm

screen shot 2017-09-13 at 2 10 51 pm

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -241,12 +241,78 @@ const filecoin = {
...
+    q3: {answer: true},
+    q4: {answer: true},
+    q5: {answer: true, comment: `Crowdsale contract provides no log event, its provided in the token contract.`},
+    q6: {answer: true},
+    q7: {answer: true},
+    q8: {answer: null},
+    q9: {answer: null},
+    q10: {answer: true},
+    q11: {answer: true},
+    q12: {answer: true},
```
Comment (MostafaBalata) [inline src/config.js]: question: 'Is price of the token controlled by smart contract?',
@rudolfix Can you check this answer?

File: src/components/details.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ export const TokenIssued = ({ tokenIssued, tokensOverflow, totalInvestors, total
             <td>{formatNumber(tokenIssued)}</td>
           </tr>
           {
-            tokensOverflow !== 0 &&
+            parseFloat(formatNumber(tokensOverflow)) !== 0 &&
```
Comment (MostafaBalata) [inline src/components/details.js]: This condition needs to be better

File: src/config.js [hunk 3]
```diff
@@ -241,12 +241,80 @@ const filecoin = {
...
+      lastTransactionBlockNumber:  4241086,
+      countTransactions: true,
+      address: '0x2c974b2d0ba1716e644c1fc59982a89ddd2ff724'
+    }
+  },
+  icoParameters: {
+    cap: async(web3, icoContract) => {    
+      const maxCap = await toPromise(icoContract.maxCap)().valueOf();
+      const minCap = await toPromise(icoContract.minCap)().valueOf();
+      return [`Max ${(maxCap / (10 ** 18)).toFixed(2)} VIB`,`Min ${(minCap / (10 ** 18)).toFixed(2)} VIB`];
```
Comment (rudolfix) [inline src/config.js]: 1. Why aren't you using `convertWeb3Value` to convert wei to ether you yourself wrote?
2. Caps are incorrect, please find problem and fix

File: src/config.js [hunk 4]
```diff
@@ -241,12 +241,80 @@ const filecoin = {
...
+};
+
+
 let config = {
   ICOs: {
     'kin-smart-contract-not-provided': kin,
     '0xd4FD252d7D2C9479a8d616F510eAC6243B5DDdf9': zrx,
     'filescoin-smart-contract-not-provided': filecoin,
     '0x1d0dcc8d8bcafa8e8502beaeef6cbd49d3affcdc': gnosis,
+    '0x91c94bee75786fbbfdcfefba1102b68f48a002f4': viberate,
```
Comment (rudolfix) [inline src/config.js]: should be below ZRX

File: src/config.js [hunk 5]
```diff
@@ -241,12 +241,80 @@ const filecoin = {
...
+    status: async icoContract => 'successful',
+  },
+  matrix: {
+    q1: {answer: true},
+    q2: {answer: true},
+    q3: {answer: true},
+    q4: {answer: true},
+    q5: {answer: true, comment: `Crowdsale contract provides no log event, its provided in the token contract.`},
+    q6: {answer: true},
+    q7: {answer: true},
```
Comment (rudolfix) [inline src/config.js]: There is transparency issue in smart contract. Let's talk

File: src/config.js [hunk 6]
```diff
@@ -241,12 +241,80 @@ const filecoin = {
...
+    logo: 'https://viberateassets.azureedge.net/favicon.ico',
+  },
+  events: {
+    Mint :{
+      args: {
+        tokens: '_value',
+        sender: '_to',
+      },
+      firstTransactionBlockNumber: 4240691,
+      lastTransactionBlockNumber:  4241086,
```
Comment (rudolfix) [inline src/config.js]: put block number that will exclude tokens created after ICO

**Comment by rudolfix** [comment]
Did you really merged BAT pull request into this? Why? This is a separate PR with separate review! Please remove commit:7f31c06 and just merge/rebase master

File: src/config.js [hunk 7]
```diff
@@ -241,6 +241,71 @@ const filecoin = {
...
+    logo: 'https://viberateassets.azureedge.net/favicon.ico',
+  },
+  events: {
+    Mint :{
+      args: {
+        tokens: '_value',
+        sender: '_to',
+      },
+      firstTransactionBlockNumber: 4240691,
+      lastTransactionBlockNumber:  4348935,
```
Comment (rudolfix) [inline src/config.js]: This is still invalid block number. Look at the results! must be block before developers claim their tokens. You may use your new tool to find this block ;>

File: src/config.js [hunk 8]
```diff
@@ -301,10 +366,12 @@ const bnt = {
   addedBy: 'Mostafa Balata',
 };
 
+
 let config = {
   ICOs: {
     'kin-smart-contract-not-provided': kin,
     '0xd4FD252d7D2C9479a8d616F510eAC6243B5DDdf9': zrx,
+    '0x91c94bee75786fbbfdcfefba1102b68f48a002f4': viberate,
```
Comment (rudolfix) [inline src/config.js]: above ZRX! we order by date more or less!

File: src/config.js [hunk 9]
```diff
@@ -301,10 +366,12 @@ const bnt = {
...
 };
 
+
 let config = {
   ICOs: {
     'kin-smart-contract-not-provided': kin,
     '0xd4FD252d7D2C9479a8d616F510eAC6243B5DDdf9': zrx,
+    '0x91c94bee75786fbbfdcfefba1102b68f48a002f4': viberate,    
     'filescoin-smart-contract-not-provided': filecoin,
     '0xBbc79794599b19274850492394004087cBf89710': bnt,
```
Comment (rudolfix) [inline src/config.js]: Did you really merged BAT pull request into this? Why? This is a separate PR with separate review! Please remove commit:7f31c06 and just merge/rebase master

**Pull request merged**